Update Activate component styles for improved readability

gonojuarez opened the pull request:
Change the background color 
```html
.Activate {
  height: 100%;
  padding: 1em;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: #eceff1;
  background-color: linear-gradient(
    to right,
    rgb(45, 22, 254),
    rgb(141, 92, 255)
  );
}
```
to

```html
.Activate {
  height: 100%;
  padding: 1em;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  color: #000;
  background-color: linear-gradient(
    to right,
    rgb(45, 22, 254),
    rgb(141, 92, 255)
  );
}
```
close #1888

gonojuarez committed:
Update Activate component styles for improved readability

tomivm reviewed (changes requested):
Hey Gon! sorry for the delay. I think the problem is in the background color and not in the color of the text

gonojuarez committed:
Update Activate component styles for better aesthetics and readability

martinbedouret committed:
Refactors translation fetching and extraction

Streamlines the process of fetching and extracting translations from Crowdin.

- Updates dependencies and configurations for improved performance.
- Modifies file handling for custom language support and error management.
- Cleans up temporary files after extraction.

RodriSanchez1 committed:
Merge branch 'master' of github.com:cboard-org/cboard

martinbedouret committed:
Merge pull request #1912 from cboard-org/fix/picseepalBoardTilesGridOrder

Fix forced push

gonojuarez committed:
Merge branch 'cboard-org:master' into bug_hotfix_error_activate
